packages/theme/middleware.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
           // 'enabled' (Spree 4.4+)
           // 'legacy' (Spree older than 4.4 with spree_wishlist gem installed)
           // 'disabled' (Spree older than 4.4 without spree_wishlist gem)
+          useMenuApi: false,
```

Comment:
I do not know if user should be able to set that property manually or it should be set automatically based on used spree version a few lines above. If it is set automatically there could be a few problems when moving from older spree to the newest ones(not properly set backend), but setting it manually may lead to some other issues. Any ideas?